pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/BooleanSchema.java

```diff
@@ -32,10 +32,10 @@ public class BooleanSchema extends AbstractSchema<Boolean> {
     private static final SchemaInfo SCHEMA_INFO;
 
     static {
-        SCHEMA_INFO = new SchemaInfoImpl()
-                .setName("Boolean")
-                .setType(SchemaType.BOOLEAN)
-                .setSchema(new byte[0]);
+        SCHEMA_INFO = SchemaInfoImpl.builder()
+                .name("Boolean")
+                .type(SchemaType.BOOLEAN)
+                .schema(new byte[0]).build();
```

Comment:
The class `SchemaInfoImpl` is a stable API, we can't change the existing API.

Please check the annotations of the class `SchemaInfoImpl`

```
@InterfaceAudience.Public
@InterfaceStability.Stable
```

Reply:
OK. I have updated to keep the `SchemaInfoImpl` be Stable. 
But this change, from `new SchemaInfoImpl()` to `SchemaInfoImpl.builder()` is necessary I think